docs/modules/ROOT/pages/includes/attributes.adoc

```diff
@@ -13,5 +14,3 @@
 :quickstarts-archive-url: https://github.com/quarkusio/quarkus-quickstarts/archive/main.zip
 :quickstarts-blob-url: https://github.com/quarkusio/quarkus-quickstarts/blob/main
 :quickstarts-tree-url: https://github.com/quarkusio/quarkus-quickstarts/tree/main
-
-:imagesdir: ../images
```

Comment:
Antora requires specific placement of the images. It's all taken care of with my stuff.